changelog.d/0-release-notes/pr-2641

```diff
@@ -0,0 +1 @@
+Deploy nginz before brig.
```

Comment:
Interesting, I just realized that this is not a problem at after all, because the new endpoint becomes available once both services are updated. So order does not matter.